### ISSUE TITLE
Accessibility Assessment of the app (for the pilot)

### ISSUE THREAD

**Issue by GuillaumeTurcotte** [dto-btn/chatbot-frontend#71]
## Digital Hour feedback

During digital hour we got feedback on the accessibility. Example: 

- I did a quick check, my keyboard cannot reach the "Clear chat" "SSC Early Adopters" and "Developer Settings" buttons, this will exclude many categories of users.

## Accessibility contact

aaact-aatia@ssc-spc.gc.ca

- [x] Meeting plan November 23rd

## Success criteria

- [x] keyboard accessible
- [x] violation fixed (those fixables)
- [ ] narrator test
- [ ] Review of [WCAG 2.1](https://www.w3.org/TR/WCAG21/#background-on-wcag-2)

**Comment by fociera** [comment]
[ICT accessibility requirements.docx](https://github.com/dto-btn/chatbot-frontend/files/13461838/ICT.accessibility.requirements.docx)

**Comment by GuillaumeTurcotte** [comment]
Some initial work was done to fix a wide variety of issue which I think are pretty good for a pilot phase: https://github.com/dto-btn/chatbot-frontend/pull/75
